### PR TITLE
Task/4 create a script to seed data

### LINKED ISSUE #4: Create the script to automatically seed the data from the scraper into the shop
Write a script (preferably use python) to seed the data (which is a result of scraper) into the PrestaShop via REST API.

Additionally, make sure that points

- 3 > f
- 3 > g
- 3 > h

are also fulfilled.

### PR DESCRIPTION
**Pull request by fildaw**
Created a script to seed data from the scraper using PrestaShop REST API and a script for removing data from the shop.
Fixed scraper to preserve formatting in product description and updated scrapped data.

Closes #4

### CODE REVIEW

File: data_seeder/seeder.py [hunk 1]
```diff
@@ -0,0 +1,303 @@
...
+urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
+
+
+DATA_DIR = Path(__file__).parent.parent / "data"
+DATA_JSON_FILENAME = "data.json"
+XML_TEMPLATES_DIRNAME = Path(__file__).parent / "xml_templates"
+
+API_KEY = os.environ["PRESTASHOP_API_KEY"]
+
+IP_PORT = "https://localhost:8443"
```
Comment (Nasus20202) [inline data_seeder/seeder.py]: Can we use an environment variable here, with localhost as a fallback value?

**Comment by Nasus20202** [comment]
Also, I noticed an issue with importing some products. Maybe we can skip them?
image

File: data_seeder/seeder.py (hunk 1 above)
Comment (fildaw) [reply data_seeder/seeder.py]: Sure

**Review by Nasus20202 — APPROVED** [review]

**Review by KanarekLife — APPROVED** [review]
lgtm

**Pull request merged**